### PR TITLE
Update tutorial.md

### PR DESCRIPTION
**Pull request by subinamehta**
Updated the PG tutorials 1,2 and 3 after receiving user feedback

### CODE REVIEW

**Comment by subinamehta** [comment]
Updated the workflow for metaproteomics and GTN material for proteogenomics

File: topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.md [hunk 1]
```diff
@@ -181,7 +186,7 @@ Once this step is completed, a tabular output containing novel proteoforms are d
...
+The Multiomics Visualization Platform is a galaxy visualization plugin that allows the user to browse the selected proteomics data. It uses the SQlite database which allows the data to be filtered and aggregated in a user defined manner. It allows various features such as; the PSM can be displayed with a lorikeet spectral view, the selected peptide can be displayed in a protein view and an IGV browser is also available for the selected protein. The step by step guide shown below will provide a walkthrough on how to use this plugin (NOTE: the example shown below is a representative peptide which is subjected to change, so while you are running this too please take a look at the "Novel Peptide" output from the previous steps).
```
Comment (shiltemann) [inline topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.md:189]:
```suggestion
The Multiomics Visualization Platform is a Galaxy visualization plugin that allows the user to browse the selected proteomics data. It uses the SQlite database which allows the data to be filtered and aggregated in a user defined manner. It allows various features such as; the PSM can be displayed with a lorikeet spectral view, the selected peptide can be displayed in a protein view and an IGV browser is also available for the selected protein. The step by step guide shown below will provide a walkthrough on how to use this plugin (NOTE: the example shown below is a representative peptide which is subjected to change, so while you are running this tool please take a look at the "Novel Peptide" output from the previous steps).
```

**Review by shiltemann — DISMISSED** [review]

**Review by shiltemann — APPROVED** [review]

**Comment by shiltemann** [comment]
Thanks @subinamehta!

**Comment by subinamehta** [comment]
> Thanks @subinamehta!

Thank u Saskia!!

**Pull request merged**